### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by haarsh24**
image
image

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                    <p>Shop the best collection of sneakers ever. </p>
+                </div>
+
+                <div class="product-summary h4 ">
+                    The Latest & Greatest Sneakers .Free shipping on your first order. Beautifully crafted collection.
+                    No questions asked returns. </div>
+
+
+                <div class="btn  buy-btn">
+                    <a class="btn h4" href="./product.html">Shop Now</a>
```
Comment (manujagobind) [inline pages/index.html:44]: Use prettier to format HTML

File: styles/utlis.css [hunk 2]
```diff
@@ -35,6 +44,41 @@
   top: 0rem;
 }
 
+
+.m3 {
+  margin: 3rem;
+}
+
+.m1 {
```
Comment (manujagobind) [inline styles/utlis.css:52]: Should these not come from component library?

File: styles/main.css [hunk 3]
```diff
@@ -1,3 +1,8 @@
 @import url("./utlis.css");
 @import url("./product.css");
-@import url("./login-signup.css");
\ No newline at end of file
+
+@import url("./login-signup.css");
+
```
Comment (manujagobind) [inline styles/main.css:5]: Remove blank lines.

File: styles/utlis.css (hunk 2 above)
Comment (haarsh24) [reply styles/utlis.css:52]: Yes, at the time of developing this, component library code was not merged. will remove this from here.

File: styles/landing-page.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+.landing-page-img {
+  width: 100%;
+}
+
+.index-title {
+  font-size: 4.5rem;
+  font-weight: 700;
+}
+.front-display-div {
```
Comment (xii-vi) [inline styles/landing-page.css:14]: use a more generic name.

File: styles/utlis.css [hunk 5]
```diff
@@ -3,6 +3,15 @@
...
+
+  --text-color: #2d3748;
+  --background-color: #ddd7e0;
+  --shadow:  11px 11px 11px #21212133; 
+}
+* {
+  color: var(--text-color);
+}
+
   --text-color:#2D3748;
```
Comment (xii-vi) [inline styles/utlis.css]: why this is outside of any class

**Review by xii-vi — APPROVED** [review]
_A few suggestions. Rest is okay to merge._

**Pull request merged**